ray-operator/controllers/ray/utils/util.go

```diff
@@ -68,6 +68,24 @@ func IsRunningAndReady(pod *corev1.Pod) bool {
 	return false
 }
 
+func CheckRouteName(s string, n string) string {
+	// 63 - (max(8,6) + 5 ) - length of namespace name + 1 => 49 - namespace name
+	// 6 to 8 char are consumed at the end with "-head-" or -worker- + 5 generated.
+	// Namespace name will be appended to form: {name}-{namespace} for first host
+	//   segment within route
+	maxLength := 49 - len(n)
+
+	if len(s) > maxLength {
+		// shorten the name
+		offset := int(math.Abs(float64(maxLength) - float64(len(s))))
+		fmt.Printf("route name is too long: len = %v, we will shorten it by offset = %v\n", len(s), offset)
```

Comment:
The RayCluster controller logger should be used instead.

Reply:
Updated to use the controller logger